x-pack/plugins/alerting/server/routes/get_rule_state.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IRouter } from 'kibana/server';
+import { schema } from '@kbn/config-schema';
+import { ILicenseState } from '../lib';
+import { RewriteResponseCase, verifyAccessAndContext } from './lib';
+import { AlertingRequestHandlerContext, BASE_ALERTING_API_PATH, AlertTaskState } from '../types';
+
+const paramSchema = schema.object({
+  id: schema.string(),
+});
+
+const rewriteBodyRes: RewriteResponseCase<AlertTaskState> = ({
+  alertTypeState,
+  alertInstances,
+  previousStartedAt,
+  ...rest
+}) => ({
+  ...rest,
+  alert_type_state: alertTypeState,
+  alert_instances: alertInstances,
```

Comment:
Should these be `rule_type_state` and `alerts` to follow the new terminology?

Reply:
Good catch! Fixed in https://github.com/elastic/kibana/pull/93980/commits/02bc3926cbb95f428d73ff011bf24f0fec19922d.